packages/bbui/src/Form/Core/TextField.svelte

```diff
@@ -13,10 +13,18 @@
   export let quiet = false
   export let dataCy
   export let align
+  export let autofocus = false
 
   const dispatch = createEventDispatcher()
+
+  let field
   let focus = false
 
+  $: if (autofocus === true && field) {
+    focus = true
+    field.focus()
+  }
```

Comment:
If we move this to a svelte `onMount` callback then we won't need to worry about whether the reference exists or not (it always will after mounting). I think semantically as well it makes sense to just explicitly focus once when mounting is complete.

Reply:
Initially this was intended to allow the value to continue to be changed after loading, but in this scenario that isn't required. The behaviour has been updated.